README.md

```diff
@@ -15,6 +15,21 @@ yarn install
 yarn install-global
 ```
 
+Due to some issues with yarn installation through npm, it may be entirely possible that the location where gradec was installed is not in your PATH environment variable. If gradec is not running after `yarn install-global`, follow these steps. 
+
+If you're running iOS/Linux:
+```bash
+yarn path-nix
+# copy the above output for use below
+export PATH=COPY_PASTE_OUTPUT_HERE
+```
+
+If you're running Windows (powershell):
+```powershell
+yarn path-ps
+# copy the above output for use below
+$env:path += COPY_PASTE_OUTPUT_HERE
```

Comment:
It's better to suggest to put this in the shell config file (like .bashrc or windows env variables). Otherwise, the updated variable will be active for the time the shell is active.